### ISSUE TITLE
left ventricle morphology terms

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3600]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

a.  increased heart left ventricle end-diastolic posterior wall thickness
b.  decreased heart left ventricle end-diastolic internal diameter

>2. Synonyms (e.g., absent spleen)

a.  increased LVPWd; increased end-diastolic heart left ventricle posterior wall thickness; increased end-diastolic heart left ventricle PWT; increased left ventricular posterior wall thickness at diastole
b  decreased LVIDd; decreased heart left ventricular end-diastolic internal diameter; decreased left ventricular internal dimension at diastole

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

a. The thickness or depth (i.e. the measurement of the two-dimensional extent of the distance between the inner and outer surfaces) of the dorsal/posterior wall of the left ventricle of the heart at the end of diastole, that is, when the heart muscle is maximally relaxed. The dorsal wall is the portion of the muscle enclosing the ventricle which is closest to the spine and farthest from the chest wall of the organism.  The left ventricle is the lower chamber of the left side of the heart, which pumps oxygenated blood out through the aorta to all the tissues of the body.
b. The diameter across the inside of the left ventricle of the heart at the end of diastole, that is, when the heart muscle is maximally relaxed, and usually corresponds to its largest diameter.

PMID:32207065

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

a. heart left ventricle hypertrophy (MP:0002625)
b. heart left ventricle hypertrophy (MP:0002625)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0003-3645-6803

**Comment by sbello** [comment]
missing eqs for some terms due to absence of Uberon terms for specific parts of the walls
